sleap/nn/callbacks.py

```
@@ -4,6 +4,7 @@
 import logging
 import numpy as np
 import tensorflow as tf
+import qtpy
```

Ruff (0.8.2)

`qtpy` imported but unused

Remove unused import: `qtpy`

(F401)